cmd/go-clang-gen/main.go

```diff
@@ -5,12 +5,12 @@ import (
 	"os"
 	"strings"
 
-	generate "github.com/go-clang/gen"
-	generateclang "github.com/go-clang/gen/clang"
+	"github.com/go-clang/gen"
+	"github.com/go-clang/gen/clang"
```

Comment:
I would prefer to see an explicit package name `genclang` here to make it clear that it is `gen/clang` and not one of our `clang` packages that is imported here.